CONTRIBUTING.md

```diff
@@ -29,23 +27,19 @@ Documentation is kept and published via a git repository:
 
 ## Workflow
 
-### JIRA and Github issues
+### Github issues
 
-The project uses a JIRA and also githb issues to track work.  Please create one
-of a JIRA issue or a github issue so that we can track a contribution.
-
-JIRA:
-
-    https://issues.apache.org/jira/browse/JENA
+The project github issues to track work.  Please create one
```

Comment:
The project uses?

Reply:
And at the end of the line; Please create a github issue maybe?

Reply:
Done